tests/installation/releasenotes.pm

```diff
@@ -5,8 +5,15 @@ use testapi;
 sub run(){
     my $self=shift;
 
-    assert_and_click 'release-notes-button';    # open release notes window
-    assert_and_click 'release-notes-tab-sle';   # click on first SLES tab
+    if (check_var('VIDEOMODE', 'text')) {
+        send_key "alt-l";   # open release notes window
+        send_key 'alt-s';   # jump to first tab
+        assert_screen 'release-notes-tab-sle';
+    }
+    else {
+        assert_and_click 'release-notes-button';    # open release notes window
+        assert_and_click 'release-notes-tab-sle';   # click on first SLES tab
+    }
```

Comment:
hmm...yes, IIRC "Help" and "Release Note" button doesn't provide the shortcut in graphical mode since SLE12-GA